test/test_argo_engine.py

```diff
@@ -457,7 +457,7 @@ def test_argo_engine_get_archived_workflow_log_succeeded():
 
 def test_argo_engine_get_workflow_log_succeeded():
     """
-    Fetch workflow details at workflow endpoint, but failed to fetch at archived workflow endpoint
+    Fetch workflow error logs at workflow endpoint, but failed to fetch at archived workflow endpoint
     """
     engine = ArgoEngine()
     mock_return_archived_wf = {"code": 5, "message": "not found"}
```

Comment:
do we need this line?

Reply:
Yup i noticed that too right when I opened the PR, just removed it